packages/lib-ts/src/types/ChainId.ts

```diff
@@ -8,4 +8,5 @@ export enum ChainId {
   ARBITRUM = 42161,
   BASE = 8453,
   GNOSIS = 100,
+  SOLANA_MAINNET = 0,
```

Comment:
Let me know if it makes sense to have this with this value currently, or we should figure something else out.

For the record, Solana clusters are identified either by their common name (i.e. `mainnet-beta` here) or their genesis hash (i.e. `5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d` here)

Reply:
After discussion, we'll go with using 0 for now. Can be changed in the future.

Reply:
After re-thinking why not having sth different than 0 to avoid having issues with zeroed values in TS? Could be any random integer